src/main/java/frc/robot/Constants.java

```diff
@@ -18,11 +20,17 @@ public final class Constants {
 
 
   public static class OperatorConstants {
-    public static final int kDriverControllerPort = 0;
+    /* Driver controller  constants*/
+    public static final int DRIVER_CONTROLLER_PORT = 0;
+
+    /* Operator Controller constants */
+    public static final int OPERATOR_CONTROLLER_PORT = 1;
+    public static final int GRIPPER_INTAKE_BTN_ID = 5;
```

Comment:
adhering to previous years' `OP_INTAKE_BTN` would be preferred: start variable with controller its on, make command name concise, and including `ID` at the end is redundant

Reply:
dont mark as resolved w/out addressing comments